data/transactions/logic/TEAL_opcodes.md

```diff
@@ -1214,74 +1214,74 @@ The notation A,B indicates that A and B are interpreted as a uint128 value, with
 ## b+
 
 - Opcode: 0xa0
-- Stack: ..., A: []byte, B: []byte &rarr; ..., []byte
+- Stack: ..., A: bigint, B: bigint &rarr; ..., bigint
```

Comment:
I assume you defined `bigint` as byte(0,64). In that case, the return type is wrong, as it may exceed that length, going up to 65 bytes.